tests/armis_sdk/core/armis_client_test.py

```diff
@@ -149,3 +149,60 @@ async def test_list_with_multiple_pages(
         {"id": "4", "name": "mock_site_4"},
         {"id": "5", "name": "mock_site_5"},
     ]
+
+
+async def test_proxy_configuration(monkeypatch, httpx_mock: pytest_httpx.HTTPXMock):
+    """Test that proxy environment variables are properly configured in the client."""
+
+    proxy_url = "http://test-proxy:8080"
+    monkeypatch.setenv("HTTPS_PROXY", proxy_url)
+
+    httpx_mock.add_response(
+        method="POST",
+        url="https://mock_tenant.armis.com/api/v1/access_token/",
+        json={"data": {"access_token": "mock_token", "expires_in": 3600}},
+    )
+
+    httpx_mock.add_response(
+        url="https://mock_tenant.armis.com/mock/endpoint", json={"success": True}
+    )
+
+    armis_client = ArmisClient()
+
+    proxy = armis_client._get_proxy_config()
+    assert proxy == proxy_url, f"Expected {proxy_url}, got {proxy}"
+
+    async with armis_client.client() as client:
+        assert (
+            client._trust_env is True
+        ), "trust_env should be True for environment variable support"
+
+        # Test that requests work with proxy configuration
+        response = await client.get("/mock/endpoint")
```

Comment:
What are we actually testing here? We don't check if the actual proxy url was accessed...